account/src/controllers/accounts.controller.js

```diff
@@ -1,47 +1,48 @@
-const AccountsModel = require('../models/accounts.model');
-const HTTPStatus = require('../helpers/HTTP.status');
-const validate = require('../validations/accounts.validations');
-
-const findAll = async (_req, res) => {
-  const response = await AccountsModel.find();
-  return res.status(HTTPStatus.OK).json(response);
-}
-
-const findOne = async (req, res) => {
-  const { id } = req.params;
-  const response = await AccountsModel.findById(id);
-
-  return res.status(HTTPStatus.OK).json(response);
-}
-
-const create = async (req, res) => {
-  const payload = req.body;
-  validate.payload(payload);
-
-  const response = await AccountsModel.create(payload);
-  return res.status(HTTPStatus.CREATED).json(response);
-}
-
-const edit = async (req, res) => {
-  const { id } = req.params;
-  const payload = req.body;
-  validate.payload(payload);
-
-  const response = await AccountsModel.findByIdAndUpdate(id, payload, { new: true });
-  return res.status(HTTPStatus.OK).json(response);
-}
-
-const deleteOne = async (req, res) => {
-  const { id } = req.params;
-
-  await AccountsModel.findByIdAndDelete(id);
-  return res.status(HTTPStatus.NO_CONTENT).end();
-}
-
-module.exports = {
-  findAll,
-  findOne,
-  create,
-  edit,
-  deleteOne,
-}
+const AccountsModel = require('../models/accounts.model');
+const HTTPStatus = require('../helpers/HTTP.status');
+const validate = require('../validations/accounts.validations');
+
+const findAll = async (_req, res) => {
+  const response = await AccountsModel.find();
+  return res.status(HTTPStatus.OK).json(response);
+};
+
+const findOne = async (req, res) => {
+  const { id } = req.params;
+  const response = await AccountsModel.findById(id);
```

Comment:
Seria interessante envolver estes blocos em um try/catch e tratar o retorno de erro e o retorno quando a entidade não for encontrada.